doc/classes/ProjectSettings.xml

```diff
@@ -3475,6 +3475,11 @@
 			Practically speaking, this means that the end result of the Viewport will not be clamped to the [code]0-1[/code] range and can be used in 3D rendering without color encoding adjustments. This allows 2D rendering to take advantage of effects requiring high dynamic range (e.g. 2D glow) as well as substantially improves the appearance of effects requiring highly detailed gradients.
 			[b]Note:[/b] This property is only read when the project starts. To toggle HDR 2D at runtime, set [member Viewport.use_hdr_2d] on the root [Viewport].
 		</member>
+		<member name="rendering/viewport/tonemap_before_blending" type="bool" setter="" getter="" default="false">
+			If [code]true[/code], enables [member Viewport.tonemap_before_blending] on the root Viewport. When enabled, post-processing options will be limited to tonemapping only and it'll be applied directly when drawing 3D geometry. This can improve performance at the cost of inaccurate blending and weaker transparencies. This is the same technique employed by the Compatibility renderer.
```

Comment:
```suggestion
			If [code]true[/code], enables [member Viewport.tonemap_before_blending] on the root Viewport. When enabled, post-processing options will be limited to tonemapping only and will be applied directly when drawing 3D geometry. This can improve performance at the cost of inaccurate blending and weaker transparencies. This is the same technique employed by the Compatibility renderer.
```
I'd say